web/packages/web-client/pages/notebook.tsx

```diff
@@ -0,0 +1,23 @@
+import { PureComponent } from 'react';
+import auth from '../libs/auth';
```

Comment:
Because of the Webpack config, auth can be imported multiple times across components, while also being included only once across the final collection of bundles.

The current Webpack config handles the mapping from components to flat bundles quite well.